CONTRIBUTING.md

```diff
@@ -1,54 +1,87 @@
 
-* "Sign-off" your commits (NOTE! This is only necessary if contributing to the vertx-core project)
+Committers may direct you to another repository, ask for additional clarifications, and add appropriate info before the issue is addressed.
 
-Every commit you make in your patch or pull request MUST be "signed off".
+For questions that are not an issue with the code per se, it is recommended that you refer to the [community group](https://groups.google.com/forum/#!forum/vertx) since it allows the whole community to help instead of just the contributors and commiters.
 
-You do this by adding the `-s` flag when you make the commit(s), e.g.
 
-    git commit -s -m "Shave the yak some more"
+# Contributions
 
-## Making your changes
+Any change to resources in this repository must be through pull requests. This applies to all changes to documentation, code, binary files, etc. Even long term committers must use pull requests.
 
-* Fork the repository on GitHub
-* Create a new branch for your changes
-* Make your changes
-* Make sure you include tests
-* Make sure the test suite passes after your changes
-* Commit your changes into that branch
-* Use descriptive and meaningful commit messages
-* If you have a lot of commits squash them into a single commit
-* Make sure you use the `-s` flag when committing as explained above.
-* Push your changes to your branch in your forked repository
+No pull request can be merged without being reviewed.
 
-## Submitting the changes
+It is expected that all contributors to Eclipse Vert.x organization sign the [Eclipse Contributor Agreement](http://www.eclipse.org/legal/ECA.php). In order to sign the ECA a contributor is required to have an Eclipse Foundation user id and read and digitally sign the following [document](http://www.eclipse.org/contribute/cla). Digitally signing is as simple as read the document and submit the "I agree" button.
 
-Submit a pull request via the normal GitHub UI.
- 
-## After submitting
+For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that contributors in other timezones have time to review. Consideration should also be given to  weekends and other holiday periods to ensure active committers all have reasonable time to become involved in the discussion and review process if they wish.
 
-* Do not use your branch for any other development, otherwise further changes that you make will be visible in the PR.
+The default for each contribution is that it is accepted once no committer has an objection. During review committers may also request that a specific contributor who is most versed in a  particular area gives a "LGTM" before the PR can be merged. There is an additional "sign off" process for contributions to land. All commits to code in the Eclipse Vert.x organization **MUST** be signed off. This is done when commiting from git passing the extra argument `-s` e.g.:
```

Comment:
Shouldn't the sentence about "signoff" be in another paragraph? I'd rather see it with the Eclipse CLA sentence.